aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -326,6 +326,86 @@ To modify the connection events, register callbacks for the following connection
 </body>
 ```
 
+### Advanced scenario: Automatically refresh the page when reconnection fails
```

Comment:
We almost never tag headings this way. Can it just call out what this content covers? ...

```suggestion
### Automatically refresh the page when reconnection fails
```